optimade/adapters/structures/aiida.py

```diff
@@ -17,9 +26,14 @@
 
 
 def get_aiida_structure_data(optimade_structure: OptimadeStructure) -> StructureData:
-    """ Get AiiDA StructureData from OPTIMADE structure
-    :param optimade_structure: OPTIMADE structure
-    :return: StructureData
+    """ Get AiiDA `StructureData` from OPTIMADE structure.
+
+    Parameters:
+        optimade_structure: OPTIMADE structure.
+
+    Returns:
+        AiiDA `StructureData` Node.
```

Comment:
Try it out, if you replace `StructureData = None` on the failed import with `StructureData = type("StructureData", (), {})` (likewise jarvis/pmg etc) then these packages don't need to be imported successfully for the type annotations to be correct

Reply:
> When building this locally the return type doesn't render, I think because aiida cannot be installed alongside the mkdocs versions we have (due to conflicting tornado versions).

Right. I always manually install `tornado==5.1.1` to make it work... :/

Reply:
> Try it out, if you replace `StructureData = None` on the failed import with `StructureData = type("StructureData", (), {})` (likewise jarvis/pmg etc) then these packages don't need to be imported successfully for the type annotations to be correct

Oohh

Reply:
Does it really make sense to use this for numpy too? I don't see it messing up the docs anywhere?

Reply:
Hmm.. maybe, maybe not. If you don't want to use it for numpy then it changes the situation as it was before this PR as well. Which means, I would have to bump `numpy` to more "required" state of dependency.

Reply:
Ah, but maybe `numpy` is not actually used in the docs anywhere for type annotations?

Reply:
Hereby reverted.